cypress/pageObjects/LoginPage.js

```diff
@@ -0,0 +1,34 @@
+export default class LoginPage {
+    _loginPageUrl = '/auth/login'
+    _landingPageUrl = '/pages/dashboard'
```

Comment:
this url is not related to login page

either create another page object or create file where you store all URLs

Reply:
Updated